src/Symfony/Component/VarExporter/Tests/Fixtures/LazyGhostObject/MagicClass.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarExporter\Tests\Fixtures\LazyGhostObject;
+
+class MagicClass
+{
+    public static $destructCounter = 0;
+    public $cloneCounter = 0;
```

Comment:
missing property types?

Reply:
types added and other comments addressed.
thanks for the review